content/en/docs/reference/command-line-tools-reference/feature-gates/prefer-align-cpus-by-uncorecache.md

```diff
@@ -0,0 +1,13 @@
+---
+title: PreferAlignCpusByUncoreCache
+content_type: feature_gate
+_build:
+  list: never
+  render: false
+
+stages:
+  - stage: alpha
+    defaultValue: false
+    fromVersion: "1.32"
+---
+When `PreferAlignCpusByUncoreCache` is enabled while the CPU Manager Policy is set to `static`, containers within a `Guaranteed` pod will individually be aligned to an uncore cache group at a best-effort policy. This feature can optimize performance for certain cache-sensitive workloads by minimizing the cpu allocation across uncore caches.
```

Comment:
Try this
```suggestion
Enable support for aligning workloads to a cache group that spans multiple cores: an _uncore_ cache group.

To learn more, read [CPU management policies](/docs/tasks/administer-cluster/cpu-management-policies/).
```

Reply:
This should ideally link to a concept page instead; see https://github.com/kubernetes/website/pull/48340#pullrequestreview-2443697490

Reply:
> See https://kubernetes.io/docs/concepts/policy/ - can you add a very brief page into that section that explains the idea?
This feature is a CPU Manager Policy (under Node Resource Managers). Just want to confirm my understanding that you are asking for a standalone concept page for `prefer-align-cpus-by-uncorecache` that is separate from the existing CPU Manager Policies concept page?

Reply:
> Just want to confirm my understanding that you are asking for a standalone concept page for `prefer-align-cpus-by-uncorecache` that is separate from the existing CPU Manager Policies concept page?


Not that, no.

I'm asking for this feature gate description to hyperlink to a concept page about CPU manager policies. The target of the hyperlink doesn't have to be specifically about `prefer-align-cpus-by-uncorecache`.

Reply:
The https://kubernetes.io/docs/concepts/policy/node-resource-managers/ concept page is _too basic_ and SIG Node should put some help towards making it more reader friendly. SIG Docs will help.

Reply:
fixed in the last upload after rebase on top of https://github.com/kubernetes/website/pull/48797